pkg/odo/cli/component/create.go

```diff
@@ -509,14 +511,19 @@ func (co *CreateOptions) Complete(name string, cmd *cobra.Command, args []string
 			}
 
 			return nil
-		} else {
+		}
+
+		if isDevfileRegistryPresent {
+			// Categorize the sections
+			log.Info("Validation")
```

Comment:
Why we add this message in each code block, if the two blocks are mutual exclusive then we should add the message globally, I test both direct mode and interactive mode it works fine for me.

Reply:
discussed with Jingfu, more info here https://github.com/openshift/odo/pull/3291#discussion_r442324400